src/Grafana.OpenTelemetry/Grafana.OpenTelemetry.csproj

```diff
@@ -16,49 +16,44 @@
 
   <ItemGroup>
     <ProjectReference Include="../Grafana.OpenTelemetry.Base/Grafana.OpenTelemetry.Base.csproj" />
-    <PackageReference Include="MinVer" Version="6.0.0">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
+    <PackageReference Include="MinVer" Version="6.0.0" PrivateAssets="all" />
   </ItemGroup>
 
   <!-- This dependency is pulled in by `OpenTelemetry.Instrumentation.Cassandra` with a minimum
        version of 3.16.0. We change this to 3.17.0, as the previous versions have dependency
        requirements that conflict with recent .NET package versions. -->
   <ItemGroup>
+    <!-- TODO Bump to 3.22.0? -->
     <PackageReference Include="CassandraCSharpDriver" Version="[3.17.0,)" />

```

Comment:
@matt-hensley Thoughts? Update to latest, or leave as is and I'll just delete the comment?